Dockerfile

```diff
@@ -1,55 +1,49 @@
-FROM registry.centos.org/centos:8
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
 LABEL \
     name="product-listings-manager" \
     vendor="product-listings-manager developers" \
     license="MIT" \
     build-date=""
 
-# https://www.centos.org/centos-linux-eol/
-RUN sed -i -e "s|mirrorlist=|#mirrorlist=|g" /etc/yum.repos.d/CentOS-* \
-    && sed -i -e "s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g" /etc/yum.repos.d/CentOS-* \
-    && yum install -y epel-release \
-    && yum -y install \
-        --setopt=install_weak_deps=false \
-        --setopt=tsflags=nodocs \
+# hadolint ignore=DL3013
+RUN microdnf -y install \
+        --setopt=install_weak_deps=0 \
+        --nodocs \
+        gcc \
         git-core \
+        krb5-devel \
+        krb5-libs \
         python3 \
-        python3-flask \
-        python3-flask-sqlalchemy \
-        python3-gunicorn \
-        python3-koji \
+        python3-devel \
         python3-pip \
         python3-psycopg2 \
-        python3-sqlalchemy
-
-RUN pip3 install \
-        flask-restful==0.3.8
+        python3-sqlalchemy \
+    && microdnf -y clean all \
+    && pip3 install --no-cache-dir \
+        flask==0.12.2 \
+        flask-restful==0.3.8 \
+        flask-sqlalchemy==2.4.4 \
+        gunicorn==20.0.4 \
+        koji
```

Comment:
It would be safer to have this only in requirements.txt and pin all versions - tox tests would ensure they work and dependabot can monitor/update it automatically.